### PR TITLE
scss colour codes cleanup by using $variables

### PR DESCRIPTION
**Pull request by mmmavis**

### CODE REVIEW

**Comment by mmmavis** [comment]
some variables come from mofo-bootstrap: https://mozilla.github.io/mofo-bootstrap/demo/

File: source/sass/main.scss [hunk 1]
```diff
@@ -254,13 +260,13 @@ button {
...
 
     cursor: pointer;
-    background: #feeb34;
+    background: $nsf-yellow;
     font-weight: bold;
     border: none;
 
     &:hover {
       border: none;
-      background: #fee601;
```
Comment (ScottDowne) [inline source/sass/main.scss]: This is a slightly darker yellow, pulled from https://healthreport.staging.wpengine.com/v01/web-literacy/

**Review by ScottDowne — CHANGES_REQUESTED** [review]
r-

**Comment by mmmavis** [comment]
Um I used the color from the Sketch file @kristinashu provided. Either the color in Sketch file is the most up to date color or it needs to be updated.

**Comment by mmmavis** [comment]
um wait @ScottDowne the Internet Health Report site **IS** using the same yellow I'm using (same as what the Sketch file uses)

screen shot 2017-02-01 at 12 23 54 pm

**Comment by ScottDowne** [comment]
The colour I have an issue with is just on the hover state, not all the colours. You're non hover state colour is correct.

**Comment by ScottDowne** [comment]
Also, the hover yellow is marginally different. It basically does nothing.

File: source/sass/main.scss (hunk 1 above)
Comment (ScottDowne) [reply source/sass/main.scss]: Not the `&:hover` bit, this is a different yellow.

**Comment by mmmavis** [comment]
@ScottDowne oh ic. PR updated.

**Review by ScottDowne — APPROVED** [review]

**Pull request merged**